pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/measurements/tests/Test_TNCuda_Expval_dev.cpp

```diff
@@ -0,0 +1,430 @@
+// Copyright 2024 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the License);
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an AS IS BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <algorithm>
+#include <complex>
+#include <limits>
+#include <memory>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#include <catch2/catch.hpp>
+
+#include "DevTag.hpp"
+#include "ExaTNCuda.hpp"
+#include "MPSTNCuda.hpp"
+#include "MeasurementsTNCuda.hpp"
+#include "TNCudaGateCache.hpp"
+#include "cuda_helpers.hpp"
+
+#include "TestHelpersTNCuda.hpp"
+
+/// @cond DEV
+namespace {
+using namespace Pennylane::LightningTensor::TNCuda::Measures;
+using namespace Pennylane::LightningTensor::TNCuda::Observables;
+using namespace Pennylane::LightningTensor::TNCuda::Util;
+} // namespace
+/// @endcond
+
+TEMPLATE_LIST_TEST_CASE("[Identity]", "[MPSTNCuda_Expval]", TestTNBackends) {
+    using TNDevice_T = TestType;
+    using Precision_T = typename TNDevice_T::PrecisionT;
+    // using Complex_T = typename TNDevice_T::ComplexT;
```

Comment:
It might be OK to remove this.

Reply:
I will remove the full file. it was just for developing :sweat_smile: